### PR TITLE
tear gas grenades

### PR DESCRIPTION
**Pull request by WindowsErrors**
last night I was in a round's dead chat and we were talking about past rounds and when I talked about how I was a sec main a long time ago, I realized this place doesn't have the wonderful tear gas grenades (simple pepperspray chem smoke grenades) that are about as ancient as this server (https://github.com/tgstation/tgstation/commit/782403ddbd3acdaf5db20d848d8fa1b771586f6b)

it's not really an upgrade or a downgrade to the typical flashbang, more like an analogue to them that's ten times as cool as a smoke bomb, I wouldn't really be worried about people abusing it, both tear gas and flashbangs make sense if you're doing an organized arrest or raid but nobody likes it when security feels the need to flashbang the entire bar because one guy is chimping out

for a real use for them, probably something like area control, most people's first instinct when they see a cloud of chemical gas moving towards them is to run the other direction down the hall, perfect for those pesky assistants piled up outside the bridge hallway, and also maybe for concealing movements or providing concealment from a shooter or something since you can't actually see through the gas

I put a box of them in the warden and hos locker and you can also get individual ones from the sectech much like with the flashbangs, as well as the xenoarchaeology replicator, right between the flashbang and smoke bomb selection

I even made an obnoxiously long box description like with the flashbang, based off the specifications sheet for the [Defense Technology Model 1063](https://www.chemicalweaponsresearch.com/munitions/military_style_saf/files/spec_sheet.pdf), although it's sadly not as long as the one for the flashbang

here you go Probey, I found this [six year old feature request](https://github.com/vgstation-coders/vgstation13/issues/6571) while I was typing this

:cl:
 * rscadd: adds tear gas grenades as security equipment, they can be found boxed in high ranking security lockers as well as individual devices in the sectech and a third special place that you'll have to look for

### CODE REVIEW

**Comment by Eneocho** [comment]
Here's hoping this one won't be powergamed (it will)

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -557,3 +557,26 @@
...
+/obj/item/weapon/grenade/chem_grenade/teargas
+	name = "teargas grenade"
+	desc = "Used for training and crowd control operations. Contents under pressure. Do not directly inhale contents."
+	stage = GRENADE_STAGE_COMPLETE
+	path = PATH_STAGE_CONTAINER_INSERTED
+	
+/obj/item/weapon/grenade/chem_grenade/teargas/New()
+	..()
+	var/obj/item/weapon/reagent_containers/glass/beaker/B1 = new(src)
+	var/obj/item/weapon/reagent_containers/glass/beaker/B2 = new(src)
```
Comment (Eneocho) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: Shouldn't these be /obj/item/weapon/reagent_containers/glass/beaker/large? Small beakers only hold 50u.

**Comment by WindowsErrors** [comment]
I have a screenshot somewhere of one of my first rounds in space station 13 when I was an assistant and I tear gassed the shuttle and everyone was coughing everywhere and vomiting and pissing and shitting and cumming in their jumpsuits for the entire shuttle ride because they couldn't escape the gas since they were on the shuttle but I couldn't find the picture but that's beside the point

**Comment by Eneocho** [comment]
If memory serves correctly, tear gas was banned in war, so using it is a war crime. Seccies now literally can commit war crimes. Fuck seccies.

**Comment by WindowsErrors** [comment]
it technically wasn't banned since tear gas isn't a poisonous gas, it's an irritant gas, which wasn't banned by the 1899 Hague convention that specified "poison or poisoned weapons"

the thing is that when the French were the first ones to use tear gas in 1914 they found that it was such a shitty weapon that the Germans didn't even detect the use of the gas at all and they thought that the mortar shells being fired were all duds

**Comment by Eneocho** [comment]
I'd reduce the amount of grenades, 7 per box is a lot, even if limited to only the HoS and Warden lockers. A good 5 would be better.

**Comment by SonixApache** [comment]
i played on goon once and they had pepperspray foam grenades that were basically area denial
wish we had those

**Comment by Eneocho** [comment]
You can make those with fluorosurfactant, condensed capsaicin and water.

**Comment by WindowsErrors** [comment]
you know I actually tested that on the testing server before I made this, it does do that, and the area of effect is much larger, about twice as large as the gas cloud

the thing with the tear foam grenade is that 1. fluorosurfactant foam is the same beige color that the cleaner grenades are and 2. fluorosurfactant foam tries to inject its chemicals (the tear foam) into people _when the foam retracts_, not when it goes over them, so they have to slip into the foam or stand in it while the foam is retracting and only when the foam disappears over them is when they actually get hit with the effects of the tear foam

so instead of denying the area to anyone who isn't wearing face protection it instead denies the area to anyone not wearing injection resistant clothing (a hardsuit)

**Comment by SonixApache** [comment]
yeah that's dumb
the ones i had on goon were chem-colored and just assblasted anyone dumb enough to walk through it without full body coverage

**Comment by Eneocho** [comment]
Best I can do is foam extinguishers with condensed capsaicin.

**Comment by GreatBigFailure** [comment]
> Best I can do is foam extinguishers with condensed capsaicin.

The best makeshift weapon

**Comment by Eneocho** [comment]
> > Best I can do is foam extinguishers with condensed capsaicin.
> 
> The best makeshift weapon

Then you haven't heard of ZAS portals or ZAS guns.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -310,6 +310,17 @@
...
...renade offers the same approximate stunning time as the Flash-Bang™ Grenade. The similar stunning times may make it the appropriate choice for training or simulation deployment of chemical agent canisters. The Tear-Gas™ formulation is considered to be less toxic than Chloral Hydrate (chloral) smoke. The Tear-Gas™ Grenade emits a very red smoke.\nIn operations, it can be utilized as a carrying agent (multiplier) for Flash-Bang™ Grenades or other stunning munitions, or for concealing the movement of security personnel. It may also be used as a distraction to focus attention away from other activities. The device should be deployed utilizing wind advantage.\nIt should NOT be deployed onto rooftops, in crawl spaces, or indoors due to its fire-producing capability. Hand throw or launch. Launching of grenades will provide deploying officers additional standoff situations.\n<B>WARNING: THIS PRODUCT IS TO BE USED ONLY BY AUTHORIZED AND TRAINED LAW ENFORCEMENT, CORRECTIONS, OR MILITARY PERSONNEL. THIS PRODUCT MAY CAUSE SERIOUS INJURY OR DEATH TO YOU OR OTHERS. THIS PRODUCT MAY CAUSE SERIOUS DAMAGE TO PROPERTY. HANDLE, STORE AND USE WITH EXTREME CARE AND CAUTION. USE ONLY AS INSTRUCTED.</B>"
```
Comment (ShiftyRail) [inline code/game/objects/items/weapons/storage/boxes.dm:316]: Bit longwinded, that one

Comment (SonixApache) [reply code/game/objects/items/weapons/storage/boxes.dm:316]: that's the case with all the grenade boxes, it's a running gag

**Comment by WindowsErrors** [comment]
i still don't regret adding this

**Pull request merged**